app/rquest-omop-worker/scripts/worker.py

```diff
@@ -102,16 +102,27 @@ def main() -> None:
         exit()
 
     logger.info("Setting up database connection...")
-    datasource_db_port = os.getenv("DATASOURCE_DB_PORT")
-    db_manager = SyncDBManager(
-        username=os.getenv("DATASOURCE_DB_USERNAME"),
-        password=os.getenv("DATASOURCE_DB_PASSWORD"),
-        host=os.getenv("DATASOURCE_DB_HOST"),
-        port=int(datasource_db_port) if datasource_db_port is not None else None,
-        database=os.getenv("DATASOURCE_DB_DATABASE"),
-        drivername=os.getenv("DATASOURCE_DB_DRIVERNAME", config.DEFAULT_DB_DRIVER),
-        schema=os.getenv("DATASOURCE_DB_SCHEMA"),
-    )
+    if bool(os.getenv("USE_TRINO", False)):
+        datasource_db_port = os.getenv("DATASOURCE_DB_PORT", 8080)
+        db_manager = TrinoDBManager(
+            username=os.getenv("DATASOURCE_DB_USERNAME", "trino-user"),
+            password=os.getenv("DATASOURCE_DB_PASSWORD"),
+            host=os.getenv("DATASOURCE_DB_HOST"),
+            port=int(datasource_db_port),
+            schema=os.getenv("DATASOURCE_DB_SCHEMA"),
+            catalog=os.getenv("DATASOURCE_DB_CATALOG")
+        )
+    else:
+        datasource_db_port = os.getenv("DATASOURCE_DB_PORT")
+        db_manager = SyncDBManager(
```

Comment:
I would expect a type error here, as `db_manager` is now allocated 2 different types. Likely solved by declaring further up:

`db_manager: TrinoDBManager | SyncDBManager`